Attester partitioning considerations (#345)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-architecture.md

### Attester Role {#attester}

In Privacy Pass, attestation is the process by which an Attester bears

> witness to, confirms, or authenticates a Client so as to verify a property
> about the Client that is required for Issuance. Issuers trust Attesters

Reviewer: ```suggestion
witness to, confirms, or authenticates a Client so as to verify properties
about the Client that are required for Issuance. Issuers trust Attesters
```

attestation procedures helps minimize the amount of information leaked through
a token.

The role of the Attester in the issuance protocol and its impact on privacy
depends on the type of attestation procedure, issuance protocol, deployment
model. For instance, requiring a conjunction of attestation procedures could
decrease the overall anonymity set size. As an example, the number of Clients
that have solved a CAPTCHA in the past day, that have a valid account, and that
are running on a trusted device is less than the number of Clients that have

> solved a CAPTCHA in the past day. Attesters SHOULD not admit attestation

Reviewer: ```suggestion
solved a CAPTCHA in the past day. Attesters SHOULD NOT be based on attestation
```

procedures that result in small anonymity sets.

> Depending on the issuance protocol, the Issuer may necessarily learn

Reviewer: ```suggestion
Depending on the issuance protocol, the Issuer might learn
```

information about the Origin. To ensure Issuer-Client unlinkability, the Issuer
should be unable to link that information to a specific Client. For such
issuance protocols where the Attester has access to Client-specific
information, such as is the case for attestation procedures that involve
Client-specific information (such as application-layer account information)
or for deployment models where the Attester learns Client-specific information
(such as Client IP addresses), Clients trust the Attester to not share any
Client-specific information with the Issuer. In deployments where the Attester
does not learn Client-specific information, the Client does not need to

> explicitly trust the Attester.

Reviewer: ```suggestion
explicitly trust the Attester in this regard.
```
